prepline_general/api/general.py

```diff
@@ -569,7 +569,7 @@ def return_content_type(filename):
 
 
 @router.post("/general/v0/general")
-@router.post(f"/general/v{app.version}/general")
+@router.post(f"/general/v0.0.49/general")
```

Comment:
in retrospect, did this need to change to be hardcoded @awalker4 ?

Reply:
I had to revert this from https://github.com/Unstructured-IO/unstructured-api/pull/256. Turns out general.py is initializing its own `app` and so this causes the endpoint to be `/general/v1.0.0/general`